added docs improvements (#139)
port-labs/port-docs: docs/exporters/github-exporter/exporting-repositories.md

    port:
      entity:
        mappings:

>           identifier: ".name" # the identifier of the entity will be the repository name, and after the creation of the entity the exporter will send `PATCH` requests to update this repository within Port

Reviewer: ```suggestion
          identifier: ".name" # The Entity identifier will be the repository name. After the creation of the Entity the exporter will send `PATCH` requests to update this repository within Port.
```

    port:
      entity:
        mappings:

>           identifier: ".name" # the identifier of the entity will be the repository name, and after the creation of the entity the exporter will send `PATCH` requests to update this repository within Port

Reviewer: ```suggestion
          identifier: ".name" # The Entity identifier will be the repository name. After the Entity is created, the exporter will send `PATCH` requests to update this repository within Port. 
```
